tests/integration/feature_manager/conftest.py

```diff
@@ -35,6 +35,5 @@ async def extended_feature_model(feature_model_dict, session, feature_store):
 
     yield feature
 
-    await session.execute_query("DELETE FROM TILE_REGISTRY")
-    await session.execute_query(f"DROP TASK IF EXISTS SHELL_TASK_{tile_id}_ONLINE")
-    await session.execute_query(f"DROP TASK IF EXISTS SHELL_TASK_{tile_id}_OFFLINE")
+    async for doc in tile_registry_service.list_documents_iterator({}):
+        await tile_registry_service.delete_document(doc["_id"])
```

Comment:
May need to check whether `doc["_id"]` is a string or `ObjectId`.

Reply:
Would it make sense for `delete_document` itself to check for this?

Reply:
We did not use `typechecked` for existing service specific method. If we add it, it may introduce some overhead.

Reply:
I am fine if we just add an assertion at the service level.

Reply:
Curious what kind of overhead might it introduce? I was thinking of simply converting the provided `document_id` to ObjectId if it's not already one.

Reply:
`typechecked` decorator internally inspect the function signature. Would prefer to some simple method like assertion or always convert the ID to `ObjectID`.